ansible/roles/master/tasks/controller-manager-configure.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: write the config file for the controller-manager
+  template: src=controller-manager.j2 dest={{ kube_config_dir }}/controller-manager
+
+- name: write the kubecfg (auth) file for controller-manager
```

Comment:
That is a problem of every task. I will fix it. All names should be upper-cased.